components/resource_metering/src/config.rs

```diff
@@ -20,9 +19,6 @@ const MIN_REPORT_RECEIVER_INTERVAL: ReadableDuration = ReadableDuration::millis(
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct Config {
-    /// Data reporting destination address.
```

Comment:
Prefer to keep it but not respect it. Otherwise for old config TiKV will throw errors.